asterius/rts/browser/default.mjs

```diff
@@ -0,0 +1,41 @@
+/**
+ * This module contains browser-specific functionality.
+ */
+export default {
+  /**
+   * The Performance Web API.
+   */
+  Performance: window.performance,
```

Comment:
The `Performance` field exported by both `browser` & `node` default script doesn't seem to be used anywhere. Also, I suggest `performance` in this script instead of explicit `window.performance`, since `performance` can also be a field of `WorkerGlobalScope`, in case the script is run inside a web worker.